### ISSUE TITLE
Update styling of the polling options and results to fulling display the content from custom poll items 

### ISSUE THREAD

**Issue by tylercopeland** [bigbluebutton/bigbluebutton#6682]
In the two screenshots below, a user might find it hard to understand which option to select without fulling seeing the text, and for the presenter, they might find it hard to parse the polling results with online showing the first four characters. My recommendation is to display the entire custom polling option(s) at all times. 

**Users Experience**

<img width="285" alt="screen shot 2019-01-29 at 6 08 09 pm" src="https://user-images.githubusercontent.com/1273169/51947610-76488500-23f3-11e9-91e2-89fe7d7036e9.png">

**Presenter Eperience**

<img width="221" alt="screen shot 2019-01-29 at 6 08 28 pm" src="https://user-images.githubusercontent.com/1273169/51947654-9c6e2500-23f3-11e9-8583-50021a1014fe.png">

**Comment by antobinary** [comment]
Regarding the Users Experience view:
@tylercopeland do you recommend expanding the buttons? From what I recall the use of Polling is to be complemented by the use of the current slide.
In such cases the polling options would be more like `A` or `A and C` or `Pears` where A, C, and the question appear on the slide in full detail. 

Regards the Presenter Experience
Yes. I can make it so that _more_ of the text would be displayed, but again, it won't be more than 12-15 chars

**Comment by Tainan404** [comment]
An idea would be to add a tool tip to each option, then if the user doesn't understand the text, he can hover the option

**Comment by tylercopeland** [comment]
@antobinary for the most part, that's the case. But, since we are also providing a custom poll type, our interface should adapt accordingly. 

What I'm thinking we can do is, if it's a custom poll type, we can have the buttons stack on top of eachother and the width of the container grows based on the length of the largest polling option. _(refer to the screenshot below)_

![screen shot 2019-02-14 at 3 51 48 pm](https://user-images.githubusercontent.com/1273169/52817178-1ac3fb80-3071-11e9-8f81-24d252a94f5d.png)


In terms of the presenter example, I think we can have the polling option text wrap.  _(refer to the screenshot below)_

![screen shot 2019-02-14 at 3 53 29 pm](https://user-images.githubusercontent.com/1273169/52817207-33341600-3071-11e9-907e-3875b2b22a58.png)
